rosa_hcp/rosa-hcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-hcp-shared-vpc-config"]
+= Configuring a shared VPC for ROSA with HCP clusters
+:context: rosa-shared-vpc-config
+
+toc::[]
+
+You can create {hcp-title-first} clusters in shared,centrally-managed AWS virtual private clouds (VPCs).
+
+[NOTE]
+====
+This process requires *two separate* AWS accounts that belong to the same AWS organization. One account functions as the VPC-owning AWS account (*VPC Owner*), while the other account creates the cluster in the cluster-creating AWS account (*Cluster Creator*).
+====
+
+image::522-shared-vpc-overview.png[]
+
+* The hosted zones can be created in either the centrally-managed VPC account or in the workload account in which the cluster is deployed.
+
+.Prerequisites for the *VPC Owner*
+* You have an AWS account with the proper permissions to create roles and share resources.
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
+* Both AWS accounts belong to the same AWS organization.
+* You enabled resource sharing from the management account for your organization.
```

Comment:
how do I do this? is it shown elsewhere? how can I tell if it's enabled/disabled?